packages/core/src/plugins/manifest.ts

```diff
@@ -25,6 +25,14 @@ type ManifestList = {
   };
   /** Flatten all assets */
   allFiles: FilePath[];
+  /** Assets grouped by chunk name */
+  namedChunks: {
```

Comment:
How about adding an option like [/webpack-manifest-plugin#generate](https://www.npmjs.com/package/webpack-manifest-plugin#generate) for customizing the manifest content?

I feel that different users may need different manifest formats, and the `generate` function can meet differentiated needs. For example, plugin-react-router can generate namedChunks through the `generate` function.

If this is feasible, I will add the `generate` option to Rsbuild soon.

Reply:
okay yeah i can add a generate option, will keep the named chunks for now, then once rsbuild api allows me to send more than true/false to it for manifest, i can move this to my plugin if you do not want to keep named chunks - however i do think named chunks is quite standard output from many stats tools as a default :P. so may be worth providing OOTB?

Reply:
> quite standard output from many stats tools as a default
Can you provide some references?

We might want to consider making `namedChunk` optional since it appears to share some overlapping information with the existing `entries` property. This could help optimize the manifest size.

Reply:
Perfect. I been working in another repo for now since I lost push access to rsbuild